cluster/ci/config/prow/config.yaml

```diff
@@ -83,6 +83,7 @@ tide:
     - openshift/machine-config-operator
     - openshift/cluster-ingress-operator
     - openshift/cluster-dns-operator
+    - openshift/cluster-api-provider-aws
     labels:
     - lgtm
     - approved
```

Comment:
If you want to require approval, you need `OWNERS` files and the `approve` plugin

Reply:
there is OWNERS in cluster-api-provider-aws repo, do I also need one here?